### ISSUE TITLE
Change branch setup, PR goes into Main

### ISSUE THREAD

**Issue by Crowdedlight** [Crowdedlight/Crows-Electronic-Warfare#68]
Remove develop and use PR straight to main. Helps with the tooling and documentation being up to date always, and avoid some merge conflicts. 

We don't need a staging which develop was as we are releasing on tags anyway.

**Comment by Crowdedlight** [comment]
Waiting for #65 and Patch to be merged into develop. Then rebaseing main on develop and delete develop.

**Comment by Crowdedlight** [comment]
After #65 is merged, and #71, #70 is merged afterwards, we can change to rename or merge into main branch and delete "develop". And do a branch cleanup in general. 
If #54 is ready, merge that before this issue, otherwise it should be fairly simple to just target "main" from that one instead of develop.
